mmdet/core/utils/misc.py

```diff
@@ -82,3 +82,24 @@ def flip_tensor(src_tensor, flip_direction):
     else:
         out_tensor = torch.flip(src_tensor, [2, 3])
     return out_tensor
+
+
+def center_of_mass(mask):
```

Comment:
Should we support the `BitMasks` in our codebase?

Reply:
Won't in this pr